helm/alfresco-content-services/templates/config-infrastructure.yaml

```diff
@@ -73,18 +73,17 @@ data:
     {{- if .Values.global.auditIndex.url }}
       {{- $elasticsearch_audit_url = .Values.global.auditIndex.url }}
     {{- else }}
-      {{- with (index .Values "elasticsearch-audit") }}
+      {{- $auditEsHost := printf "%s-%s" (.Release.Name | default "acs") (.Values.global.elasticsearch.service.name | default "elasticsearch") }}
```

Comment:
that default (`acs`) looks like a pipeline specific. I don't think we need to default `Release.Name`

I have the same remark that I think it should remain inside the `if .enabled` statement below